config.default

```diff
@@ -286,11 +286,7 @@ movingAveragePoints = 12
 # compare to observations?
 compareWithObservations = True
 # list of region indices to plot from the region list in [regions] below
-regionIndicesToPlot = [6]
-# Number of points over which to compute moving average (e.g., for monthly
-# output, movingAveragePoints=12 corresponds to a 12-month moving average
-# window)
-movingAveragePoints = 12
+regionIndex = 5
```

Comment:
@milenaveneziani, my only hesitation about the rename is that I think it would get missed in an option like 'all_timeSeries' and would be the odd analysis out.  But I may be may off base here